### PR TITLE
Added export all functionality for admin.

### PR DESCRIPTION
**Pull request by tahirs95**

### CODE REVIEW

File: pepys_admin/admin_cli.py [hunk 1]
```diff
@@ -25,7 +26,7 @@ def create_postgres_data_store():
         db_password="postgres",
         db_host="localhost",
         db_port=5432,
```
Comment (IanMayo) [inline pepys_admin/admin_cli.py]: could we keep the previous database name?

File: pepys_admin/admin_cli.py [hunk 2]
```diff
@@ -148,6 +150,40 @@ def do_export(self, arg):
...
+                    "Please provide folder name (Press Enter for auto generated folder):"
+                )
+                if folder_name:
+                    if os.path.isdir(folder_name):
+                        print("{} already exists.\n".format(folder_name))
+                    else:
+                        os.mkdir(folder_name)
+                        break
+                else:
+                    folder_name = datetime.datetime.now().strftime(
```
Comment (IanMayo) [inline pepys_admin/admin_cli.py]: we can't have ":" in pathnames for all operating systems. Happy with 8-digit date then 6-digit time `20200320_1855`

File: pepys_admin/admin_cli.py (hunk 1 above)
Comment (tahirs95) [reply pepys_admin/admin_cli.py]: Sure.

File: pepys_admin/admin_cli.py (hunk 2 above)
Comment (tahirs95) [reply pepys_admin/admin_cli.py]: Fixed.

**Pull request merged**